.travis.yml

```diff
@@ -15,7 +15,10 @@ env:
   global:
     - FLASK_APP="$(readlink -f manage.py)"
     - SQLALCHEMY_DATABASE_TEST_URI='postgresql://postgres:@localhost/portal_unit_tests'
```

Comment:
Might be a good idea to build `SQLALCHEMY_DATABASE_TEST_URI` from its constituent parts defined below, just to keep 'em in sync...